/microRTPS_agent:
  ros__parameters:
    debug: true
    receiver_cpu: 0
    sender_cpu: 1
    transport_type: UDP
    uart_baudrate: 921600
    uart_device: /dev/ttyACM0
    uart_hw_flow_control: false
    uart_polling_interval: -1
    uart_sw_flow_control: false
    udp_inbound_port: 2020
    udp_ip_address: 127.0.0.1
    udp_outbound_port: 2019
    use_sim_time: false
